src/routes/choice-exercises/__tests__/WordChoiceExerciseScreen.spec.tsx

```diff
@@ -79,41 +90,26 @@ describe('WordChoiceExerciseScreen', () => {
   it('should allow to skip an exercise and try it out later', () => {
     mockUseLoadFromEndpointWitData(testDocuments)
 
-    const { getByText } = render(<WordChoiceExerciseScreen route={route} navigation={navigation} />, {
+    const { getByText, queryByText } = render(<WordChoiceExerciseScreen route={route} navigation={navigation} />, {
       wrapper: wrapWithTheme
     })
 
     const tryLater = getByText(labels.exercises.tryLater)
     fireEvent.press(tryLater)
-    expect(getByText('Arbeitsschuhe (Plural)')).not.toBeNull()
-    fireEvent(getByText('Arbeitsschuhe (Plural)'), 'pressOut')
-    fireEvent.press(getByText(labels.exercises.next))
-    expect(getByText('Arbeitshose')).not.toBeNull()
-    fireEvent(getByText('Arbeitshose'), 'pressOut')
-    fireEvent.press(getByText(labels.exercises.next))
-    expect(getByText('Arbeitsjacke')).not.toBeNull()
-    fireEvent(getByText('Arbeitsjacke'), 'pressOut')
-    fireEvent.press(getByText(labels.exercises.next))
-    expect(getByText('Arbeitskleidung (Plural)')).not.toBeNull()
-    fireEvent(getByText('Arbeitskleidung (Plural)'), 'pressOut')
-    expect(getByText('Arbeitshose')).not.toBeNull()
-    fireEvent(getByText('Arbeitshose'), 'pressOut')
-    fireEvent.press(getByText(labels.exercises.showResults))
-  })
 
-  it('should not allow to skip last document', () => {
-    mockUseLoadFromEndpointWitData(testDocuments)
-    const { queryByText, getByText } = render(<WordChoiceExerciseScreen route={route} navigation={navigation} />, {
-      wrapper: wrapWithTheme
-    })
-
-    expect(queryByText(labels.exercises.tryLater)).not.toBeNull()
-    fireEvent(getByText('Arbeitshose'), 'pressOut')
+    let correctAnswer = getByText('Helm')
```

Comment:
You could just inline it to avoid defining a constant at all

Reply:
yeah indeed, but i think the variable makes the test better understandable, cause it is clear always the correctAnswer is clicked.